IsTheMartin/mrtnmrls
# CData fix in `]]>`

## Summary

There is an issue with `<CDATA[[]]>` in rss readers, the `]]>` is visible in the end of the article. To fix this, I had to add a break line after `[[` and before `]]`.

Also, I changed author attribute value to include email + name.

## Concern Information

To test please follow the next steps:

1. Navigate to `rss.xml` page
2. Observe `<description>` and `<author>`
